### PR TITLE
OSDOCS9297: Using /dev/fuse to access faster builds

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.15

Issue: [OSDOCS-9297](https://issues.redhat.com/browse/OSDOCS-9297), [OSDOCS-9298](https://issues.redhat.com/browse/OSDOCS-9298), [OSDOCS-9299](https://issues.redhat.com//browse/OSDOCS-9299)

Link to docs preview: https://71401--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/containers/nodes-containers-dev-fuse

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nodes-containers-dev-fuse-configuring.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-containers-dev-fuse.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-containers-dev-fuse-configuring_{context}"]
+= Configuring /dev/fuse on unprivileged pods
+
+As an alternative to the virtual filesystem, you can configure the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation to access faster builds within unprivileged pods. Using `/dev/fuse` is secure, efficient, and scalable and allows unprivileged users to mount an overlay filesystem as if the unprivileged pod was privileged.
```
Comment (adellape) [inline modules/nodes-containers-dev-fuse-configuring.adoc:9]:
```suggestion
As an alternative to the virtual filesystem, you can configure the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation to access faster builds within unprivileged pods. Using `/dev/fuse` is secure, efficient, and scalable, and allows unprivileged users to mount an overlay filesystem as if the unprivileged pod was privileged. 
```

**Comment by haircommander** [comment]
I missed this in our chat, but it could be useful to show how a user *would* do the build. something like 
```
$ oc exec -ti no-priv -- /bin/bash
$ cat >> Dockerfile <<EOF
FROM registry.access.redhat.com/ubi9
EOF
$ podman build .
```
or something

File: modules/nodes-containers-dev-fuse-configuring.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+----
++
+For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: no-priv
```
Comment (haircommander) [inline modules/nodes-containers-dev-fuse-configuring.adoc:27]: maybe rename this to `podman-pod` or something?

File: modules/nodes-containers-dev-fuse-configuring.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  annotations:
+    io.kubernetes.cri-o.Devices: "/dev/fuse"
+spec:
+  containers:
+  - name: no-priv
```
Comment (haircommander) [inline modules/nodes-containers-dev-fuse-configuring.adoc:32]: and this to `podman-container`

**Comment by DCChadwick** [comment]
@arnaik-rh this is ready for QE review.

**Comment by arnaik-rh** [comment]
@sunilcio PTAL

File: modules/nodes-containers-dev-fuse-configuring.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+[id="nodes-containers-dev-fuse-configuring_{context}"]
+= Configuring /dev/fuse on unprivileged pods
+
+As an alternative to the virtual filesystem, you can configure the `/dev/fuse` device to the `io.kubernetes.cri-o.Devices` annotation to access faster builds within unprivileged pods. Using `/dev/fuse` is secure, efficient, and scalable, and allows unprivileged users to mount an overlay filesystem as if the unprivileged pod was privileged. 
+
+.Procedure
+
+. Create the pod.
++
+[source,terminal]
```
Comment (bscott-rh) [inline modules/nodes-containers-dev-fuse-configuring.adoc:15]: Per the [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#single-command-per-code-block) please separate each command into its own code block/step.

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**